src/components/StyledFieldset.jsx

```diff
@@ -79,6 +79,23 @@ const FieldsetStyles = styled.fieldset`
   ol {
     list-style-position: inside;
   }
-`;
+
+  #registerh1, #registerh2 {
+    font-weight: bold;
+    padding: 2rem;
```

Comment:
This padding is causing the h1 to not be left-aligned:

<img width="711" alt="Screen Shot 2021-08-12 at 3 34 35 PM" src="https://user-images.githubusercontent.com/9326234/129265943-6af38089-2140-413f-a3eb-7c29fef64bf6.png">